packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.types.ts

```diff
@@ -12,6 +12,11 @@ export interface RadioGroupState extends RadioGroupProps {
    */
   value: string | null;
 
+  /**
+   * Layout of the Radios. True if horizontal or horizontal-stacked.
+   */
+  isHorizontal?: boolean;
```

Comment:
Nit: Doesn't necessarily need to be a state, can be derived from props

Reply:
Moved logic to lookup function